csharp/ql/src/Security Features/CWE-327/InsecureSQLConnection.ql

```diff
@@ -1,17 +1,14 @@
 /**
  * @name Insecure SQL connection
- * @description Using an SQL Server connection without enforcing encryption is a security vulnerability.
+ * @description Using a SQL Server connection without enforcing encryption is a security vulnerability.
```

Comment:
Depends on whether you pronounce it "S-Q-L" or "sequel" :-)

Reply:
"An SQL" is used in the ISO standard, not "a SQL": https://www.iso.org/standard/53681.html.